src/gsy_e/models/leaves.py

```diff
@@ -232,19 +238,25 @@ class ForecastSCMPV(CoefficientLeaf):
     strategy_type = ForecastSCMPVStrategy
 
 
+class ForecastSCMHeatPump(CoefficientLeaf):
+    strategy_type = ForecastScmHeatPumpStrategy
```

Comment:
I think that we discussed it some time ago too, but I do not think that we will need Forecast strategies for the SCM. Even now, we only read from profiles from the database even for Canary Networks, therefore we do not take advantage of the Redis API that is invoked with the Forecast strategies. I would suggest to remove this strategy completely, and if you agree I will add another story to remove the rest of the ForecastScm strategies.

Reply:
Agreed. Will do. Thanks for adding the ticket!